src/EditorFeatures/CSharpTest/CodeActions/InlineTemporary/InlineTemporaryTests.cs

```diff
@@ -3989,6 +3989,33 @@ class C
 {
     public void M()
     {
+        (((int, string))((1, ""hello""))).ToString();
```

Comment:
Just to check.  But *ideally* we would not need this cast right?  @DustinCampbell to see what we'd need to do in the simplifier to make this work properly.